components/molecules/BenefitCardHeaderSummary.js

```diff
@@ -7,18 +7,20 @@ export default function BenefitCardHeaderSummary(props) {
   const typesWithLinks = [SummaryTypes.PaymentAmount, SummaryTypes.LatestStatus]
 
   return (
-    <div>
-      <p>{t[props.summary.type].title}</p>
-      <p className="font-bold">{props.summary.value}</p>
-      <p className="font-bold">{props.summary.status ?? null}</p>
-      {!typesWithLinks.find((t) => t === props.summary.type) ? null : (
-        <p>
-          <Link
-            text={t[props.summary.type].linkText}
-            href={t[props.summary.type].link}
-          />
-        </p>
-      )}
+    <div className="w-full ">
+      <div className="w-full  ">
```

Comment:
There's a duplicated div here.

Reply:
Yes, I saw that and i will update. Thanks